.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "./contrib/zip"]
+	path = ./contrib/zip
+	url = https://github.com/kuba--/zip.git
```

Comment:
```suggestion
[submodule "contrib/zip"]
	path = contrib/zip
	url = https://github.com/kuba--/zip.git
```
Seems like the '.' might be throwing it off? I tried to regenerate the submodule locally, and it worked with `git submodule status`, where the cloned version of this branch did not (at least on OSX and gave the same error as the CI).

Reply:
Ping @kimkulling if you want to revive this PR.